living_documentation_regime/model/markdown_output_factory.py

```diff
@@ -0,0 +1,514 @@
+# Copyright 2025 ABSA Group Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+This module contains the Markdown Output Factory class which is responsible
+for generating outputs in the Markdown format.
+"""
+
+import logging
+import os
+from datetime import datetime
+from typing import Optional
+
+from model.output_factory import OutputFactory
+from living_documentation_regime.action_inputs import ActionInputs
+from living_documentation_regime.model.consolidated_issue import ConsolidatedIssue
+from utils.utils import make_absolute_path, generate_root_level_index_page, load_template
+from utils.constants import (
+    LIV_DOC_OUTPUT_PATH,
+    REPORT_PAGE_HEADER,
+    OUTPUT_PATH,
+    TABLE_HEADER_WITH_PROJECT_DATA,
+    TABLE_HEADER_WITHOUT_PROJECT_DATA,
+    LINKED_TO_PROJECT_TRUE,
+    LINKED_TO_PROJECT_FALSE,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class MarkdownOutputFactory(OutputFactory):
```

Comment:
This class should provide the static method which will be returning instance of required type of Exporter:
- mdoc
- pdf

No real exporting logic expected here.

Reply:
Check new way of exporting logic at commit: [7d0730a](https://github.com/AbsaOSS/living-doc-generator/pull/83/commits/7d0730ae172d3c2d228a45da9329943bad93ce20).